library/src/main/java/com/tr4android/recyclerviewslideitem/SwipeConfiguration.java

```diff
@@ -528,6 +566,10 @@ public SwipeConfiguration build() {
             config.setRightUndoDescription(mRightUndoDescription);
             config.setLeftUndoButtonText(mLeftUndoButtonText);
             config.setRightUndoButtonText(mRightUndoButtonText);
+            config.setLeftUndoDuration(mLeftUndoDuration);
+            config.setRightUndoDuration(mRightUndoDuration);
+            config.setRightUndoButtonText(mRightUndoButtonText);
```

Comment:
@aardouin This line is duplicated (check line 568, 3 lines above this one). Please delete it, thanks!